consumer/core/src/main/java/org/arquillian/pact/consumer/core/AbstractConsumerPactTest.java

```diff
@@ -19,11 +20,17 @@
 import org.jboss.arquillian.test.spi.event.suite.Test;
 
 import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
 
 public abstract class AbstractConsumerPactTest {
 
+    private static final Logger logger = Logger.getLogger(AbstractConsumerPactTest.class.getName());
```

Comment:
Not sure which convention we are following (and honestly this should be captured as part of the PR check if we agree on sth) - but if we follow the Sun convention this should be `LOGGER` instead.